backend/src/main/java/com/votogether/domain/post/controller/PostController.java

```diff
@@ -61,6 +62,18 @@ public ResponseEntity<Void> save(
         return ResponseEntity.created(URI.create("/posts/" + postId)).build();
     }
 
+    @Operation(summary = "게시글 삭제", description = "게시글을 삭제한다.")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "204", description = "게시글이 삭제되었습니다."),
+            @ApiResponse(responseCode = "400", description = "잘못된 입력입니다."),
+            @ApiResponse(responseCode = "500", description = "인터넷 서버 오류입니다.")
+    })
+    @DeleteMapping("/{id}")
+    public ResponseEntity<Void> delete(@PathVariable final Long id) {
+        postService.deleteById(id);
```

Comment:
```suggestion
        postService.deletePostById(id);
```
`P3`
삭제되는 대상을 알려주는 메서드명이면 더욱 직관적일 것 같은데 아벨은 어떻게 생각하시나요?!

Reply:
다즐의 코멘트에 추가적인 의견 남겨봅니다!
저는 개인적으로 postService라는 것에서 Post를 삭제함을 쉽게 알 수 있다는 생각이 듭니다.
하지만 PostService에서 post뿐만 아니라 다른 객체들에 대한 CRUD, 그 이상을 진행하게 된다면 혼동의 여지가 있으므로 그 때 추가하는 방법도 좋다고 생각합니다.
이에 대해 아벨은 어떻게 생각하시나요?